core/conversion/converters/impl/batch_norm.cpp

```diff
@@ -15,7 +15,7 @@ auto batch_norm_registrations TRTORCH_UNUSED = RegisterNodeConversionPatterns()
                             Tensor? mean, Tensor? var,
                             bool training, float momentum, float eps, bool cudnn_enabled) -> (Tensor))SIG",
     [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
-      auto input = args[0].ITensor();
+      auto input = args[0].ITensorOrFreeze(ctx, n);
```

Comment:
I dont think there would be a case where there is a static tensor we are going to run through batch norm, can you think of one? On the other hand is there a downside to having this?

Reply:
Yes, unlikely that a static tensor is passed in. Only case I can think of is if someone creates a non-random tensor in PyTorch and applies batch norm on it (unlikely, though). 

I added this in because for consistency's sake, there are no downsides to having this in. Shall I remove or keep it?

Reply:
I think the way I see it if they pass us a static tensor we should just do the full batch norm calculation right there since it would improve performance, I kind of want to leave it as is for layers that should not be taking in static tensors and if it is the case that we get bug reports that some are getting passed in then we should go ahead and do the computation at compile time. Maybe leave a comment that says if static tensors are getting passed in do the full calculation in the converter or something to that effect.